ui/src/app/(notary)/certificate_authorities/table.tsx

```diff
@@ -0,0 +1,373 @@
+                      {isCSRContentVisible ? "Hide CSR Content" : "Show CSR Content"}
+                    </Button>
+                    <Button
+                      className="p-contextual-menu__link"
+                      onMouseDown={() => handleCopy(caEntry.csr, caEntry.id)}>
+                      Copy CSR to Clipboard
+                    </Button>
+                    <Button
+                      className="p-contextual-menu__link"
+                      onMouseDown={() => handleDownload(caEntry.csr, caEntry.id, csrObj)}>
+                      Download CSR
+                    </Button>
+                  </span>
+                }
+                <span className="p-contextual-menu__group">
+                  <Button
+                    className="p-contextual-menu__link"
+                    disabled={caEntry.status == "pending"}
+                    onMouseDown={() => handleExpand(caEntry.id, 'Cert')}>
+                    {isCertContentVisible ? "Hide Certificate Content" : "Show Certificate Content"}
+                  </Button>
+                  {!isSelfSigned &&
+                    <Button
+                      className="p-contextual-menu__link"
+                      disabled={auth.activeCA == null}
+                      onMouseDown={() => handleSign(caEntry.id)}>
+                      {caEntry.status === "active" ? "Re-sign CSR" : "Sign CSR"}
+                    </Button>
+                  }
+                  {isSelfSigned &&
+                    <Button
+                      className="p-contextual-menu__link"
+                      onMouseDown={() => handleSign(caEntry.id, caEntry.id)}>
+                      Renew Certificate
+                    </Button>
+                  }
+                  {!isSelfSigned &&
+                    <Button
+                      className="p-contextual-menu__link"
+                      onMouseDown={() => {
+                        setCertificateFormOpen(true);
+                        setSelectedCA(caEntry);
+                      }}>
+                      Upload New Certificate
+                    </Button>
+                  }
+                  {!isSelfSigned &&
+                    <Button
+                      className="p-contextual-menu__link"
+                      disabled={caEntry.status == "pending"}
+                      onMouseDown={() => handleRevoke(caEntry.id)}>
+                      Revoke Certificate
+                    </Button>
+                  }
+                </span>
+                <span className="p-contextual-menu__group">
+                  {caEntry.status === "active" &&
+                    <Button
+                      className="p-contextual-menu__link"
+                      disabled={caEntry.status != "active" || auth.activeCA?.id === caEntry.id}
+                      onMouseDown={() => {
+                        auth.setActiveCA(caEntry);
+                      }}>
+                      Select CA for Signing
```

Comment:
I'm not sure this is clear enough that this is going to be the one and only CA signing CSR by Notary.
Maybe we could use something like:
"Select as Notary's signing CA" ?
Not sure.